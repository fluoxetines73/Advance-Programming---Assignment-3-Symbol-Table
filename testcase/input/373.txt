INSERT f_FF string
BEGIN
INSERT z string
BEGIN
BEGIN
BEGIN
LOOKUP z
BEGIN
ASSIGN z 'Test123'
LOOKUP f_FF
RPRINT
END
INSERT e_123_2r string
LOOKUP z
ASSIGN e_123_2r '1234ABC'
ASSIGN z 'UPPERlower'
LOOKUP f_FF
INSERT z string
RPRINT
LOOKUP f_FF
RPRINT
LOOKUP z
INSERT f_FF number
RPRINT
RPRINT
RPRINT
END
END
END
END