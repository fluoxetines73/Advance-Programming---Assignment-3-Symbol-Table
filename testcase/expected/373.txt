success
success
1
success
0
z//1 f_FF//0
success
1
success
success
0
success
z//4 e_123_2r//4 f_FF//0
0
z//4 e_123_2r//4 f_FF//0
4
success
f_FF//4 z//4 e_123_2r//4
f_FF//4 z//4 e_123_2r//4
f_FF//4 z//4 e_123_2r//4